tasks/service_systemd_tomcat.yml

```diff
@@ -0,0 +1,50 @@
+---
+# tasks file for universal-tomcat - installation of the service tomcat (only in recent version)
+
+- name: Install Service
+  register: __tomcat_service
+  template:
+    src: tomcat.service.j2
+    dest: /etc/systemd/system/tomcat.service
+    mode: 0644
+
+- name: Start Service
+  register: __tomcat_start_service
+  systemd:
```

Comment:
As I stated above, if this runs every time I believe this will fail the role on a system without systemd.